### DISCUSSION TITLE
blank page 404 error after deploying react project

### DISCUSSION THREAD

**Discussion by davidphamm** [community/community#28509] [Other Feature Feedback, Questions, & Ideas]
EDIT: SOLVED MY PROBLEM. I TYPED THE HOMEPAGE WRONG IN THE PACKAGE.JSON

Hello, I need help with uploading my react project to github. It shows a blank page after I deployed it.
https://github.com/davidphamm/weatherapp - the repo
https://www.imdavidpham.com/weatherapp/ - the link to open

error i been getting on console
GET https://www.imdavidpham.com/davidphamm/weatherapp/static/js/main.e398ee03.js net::ERR_ABORTED 404
GET https://www.imdavidpham.com/davidphamm/weatherapp/static/css/main.1c7ab58e.css net::ERR_ABORTED 404
GET https://www.imdavidpham.com/davidphamm/weatherapp/manifest.json 404
Manifest: Line: 1, column: 1, Syntax error.

couldn't figure out why can anyone help me?

**Comment by radar-blip** [comment]
I think you might need to include an index.html file in your repository to get that to work correctly. I could be wrong though.

**Comment by ng-druid** [comment]
I don't see the build files in that repo. 

To host a static app requires the build files either be part of that repo under a folder like dist or separate repo entirely. We have opted for the [ladder](https://github.com/rollthecloudinc/ipe-build). The repo will also need to contain an empty .jekyll file to circumvent jekyll for your own static app.

**Comment by noah-mydn** [comment]
I'm having the same issues as well. May I know how you solved it? I think I wrote the correct homepage for package.json, but still it doesn't seem to resolve the issue.
Here's my [Repo](https://github.com/noah-mydn/My-Portfolio) and [LiveDemo](https://noah-mydn.github.io/My-Portfolio/)

**Comment by ibondrk** [comment]
I'm so appreciated for your answer, man. I spent at least 3 hour to find where was my fall.

**Comment by AndroidDoctorr** [comment]
If you're still trying to figure this one out, I may have an answer. I was getting the same thing, and it was because I was missing a) a 404.html and b) I had some references that were reading as absolute rather than relative, e.g.:
'/assets/image.png' vs './assets/image.png'
